metaflow/plugins/cards/card_decorator.py

```diff
@@ -246,12 +269,14 @@ def task_finished(
             self.card_creator.create(mode="render", final=True, **create_options)
             self.card_creator.create(mode="refresh", final=True, **create_options)
 
-        # Unlink the config file if it exists
-        if self._config_file_name:
-            try:
-                os.unlink(self._config_file_name)
-            except Exception as e:
-                pass
```

Comment:
Earlier we were naively removing the config file when the same config file is referenced for all CardCreators. So this broke configs when many cards are present.